versioned_docs/version-v2.18.x/user-guide/authenticating-with-client-certificates.md

```diff
@@ -52,7 +52,18 @@ Register the client certificate with the user ID in your ESM. The following comm
 
   **Example command in RACF:**  
 
-  `RACDCERT ADD(<dataset>) ID(<userid>) WITHLABEL('<label>') TRUST` 
+  ``` 
+  RACDCERT ADD(<dataset>) ID(<userid>) WITHLABEL('<label>') TRUST
+  SETROPTS RACLIST(DIGTCERT, DIGTRING) REFRESH
+  ```
+  Alternatively, in case you are using the internal API ML mapper, you can use the following command:
+  
+  ```
+  RACDCERT ID(<userid>) MAP 
+  SDNFILTER('<subject's-distinguished-name-filter>')
+  WITHLABEL('<label>')
+  ```
+  **Note:**  The `RACDCERT MAP` command allows mapping a certificate to a user without adding the X.509 certificate to the ESM database. While this approach is more convenient, it provides less security compared to the `RACDCERT ADD` command, which stores the certificate directly in RACF, offering better control and protection.
```

Comment:
I also think we should add the commands for TSS and ACF2 in one PR, and the note should be global to all three.

```suggestion
  ``` 
  RACDCERT ADD(<dataset>) ID(<userid>) WITHLABEL('<label>') TRUST
  SETROPTS RACLIST(DIGTCERT, DIGTRING) REFRESH
  ```

  Alternatively, in case you are using the internal API ML mapper, you can use the following command:
  
  ```
  RACDCERT ID(<userid>) MAP 
  SDNFILTER('<subject's-distinguished-name-filter>')
  WITHLABEL('<label>')
  ```

  **Note:**  The `RACDCERT MAP` command allows mapping a certificate to a user without adding the X.509 certificate to the ESM database. While this approach is more convenient, it could be considered less secure than adding the certificate to the ACID as it offers better control and protection.
```

Reply:
We can keep this PR open until we finish working on the other 2 ESMs